Mixpanel/Mixpanel.m

```diff
@@ -888,11 +884,14 @@ - (void)unarchive
     [self unarchiveEventBindings];
 }
 
-- (id)unarchiveFromFile:(NSString *)filePath
+- (id)unarchiveFromFile:(NSString *)filePath asClass:(Class)class
```

Comment:
Could you just modify this to return `[class alloc] init]` of the unarchived data is nil? This would prevent hiding the ternary operator after each load call.